src/gluonts/dataset/repository/_zenodo.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import os
+from pathlib import Path
+from typing import List, Dict
+from urllib import request
+from zipfile import ZipFile
+
+from ._tsf_reader import TSFReader
+from ._util import metadata, to_dict, save_to_file
+
+from gluonts import json
+from gluonts.dataset.jsonl import dump
```

Comment:
I would prefer to do:

```py
from gluonts.dataset import jsonl
...
jsonl.dump(...)
```

Makes it clearer when it's used.

Reply:
Done